projects/components/src/filtering/filter/filter-operators.ts

```diff
@@ -9,6 +9,7 @@ export const enum FilterOperator {
   GreaterThanOrEqualTo = '>=',
   Like = '~',
   In = 'IN',
+  NotIn = 'NOT_IN',
```

Comment:
Pretty sure this isn't supported, if it's being added please put in a separate PR

Reply:
I saw one operator in Gql. So thought it is supported. 

yes, this is just a draft PR. I will create smaller ones when it is ready